src/components/Todo/Todo.tsx

```diff
@@ -25,7 +25,12 @@ const TodoComponent: React.FC<Props> = ({
   const [hasSpinner, setHasSpinner] = useState(false);
   const [isEditing, setIsEditing] = useState(false);
 
-  let isActive = todo.completed && deletingCompleted ? true : hasSpinner;
+  const activeBecauseDeletingCompleted = todo.completed && deletingCompleted;
+  let isActive = useMemo(
```

Comment:
const

Reply:
how can i use const if i cnage it in the next if block???

Reply:
you don't need to create variable with useMemo and change it in next line 
add logic inside useMemo